landing_page/spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+    context "create a new user account with acceptable credentials" do
```

Comment:
Could add tests for other controller methods too like update show etc

Reply:
Cool, I'll get onto that now 👍

Reply:
When testing a unit of code its useful to look at what the public methods are, as each warrant at least one unit test